pkg/utils/utils.go

```diff
@@ -0,0 +1,78 @@
+package utils
+
+import (
+	"fmt"
+	"net"
+	"os"
+	"strings"
+
+	multiv1beta1 "github.com/k8snetworkplumbingwg/multi-networkpolicy/pkg/apis/k8s.cni.cncf.io/v1beta1"
+	v1 "k8s.io/api/core/v1"
+)
+
+// PolicyNetworkAnnotation is annotation for multiNetworkPolicy,
+// to specify which networks(i.e. net-attach-def) are the targets
+// of the policy
+const PolicyNetworkAnnotation = "k8s.v1.cni.cncf.io/policy-for"
+
+// CheckNodeNameIdentical checks both strings point a same node
+// it just checks hostname without domain
+func CheckNodeNameIdentical(s1, s2 string) bool {
+	return strings.Split(s1, ".")[0] == strings.Split(s2, ".")[0]
+}
+
+// IsMultiNetworkpolicyTarget checks if pod is in running phase and is not hostNetwork
```

Comment:
should the and change to or?

Reply:
it returns true if both conditions hold. (is running and is not host network) so the description is correct imo